pallets/gear/src/tests.rs

```diff
@@ -701,6 +701,46 @@ fn reply_deposit_gstd_async() {
     });
 }
 
+// TODO (#2763): resolve panic caused by "duplicate" wake in message A
+#[test]
+#[should_panic]
+fn pseudo_duplicate_wake() {
+    use demo_constructor::{Calls, Scheme};
+
+    init_logger();
+    new_test_ext().execute_with(|| {
+        let (_init_mid, constructor) = init_constructor(Scheme::empty());
+
+        let execute = |calls: Calls| {
+            assert_ok!(Gear::send_message(
+                RuntimeOrigin::signed(USER_1),
+                constructor,
+                calls.encode(),
+                BlockGasLimitOf::<Test>::get(),
+                0,
+            ));
+            let mid = get_last_message_id();
+            run_to_next_block(None);
+
+            mid
```

Comment:
`mid` is too short for me, I'd use `msg_id`.

Reply:
We use such abbreviation across the project, together with pid (program_id). Furthermore, it literally "expanded" in function name - `*message_id`

Reply:
Nvm, changed to `*msg_id`